raiden/blockchain_events_handler.py

```diff
@@ -244,7 +245,7 @@ def on_blockchain_event(raiden, event, current_block_number):
         handle_tokennetwork_new(raiden, event, current_block_number)
 
     elif data['event'] == EVENT_CHANNEL_OPENED:
-        handle_channel_new(raiden, event, current_block_number)
+        handle_channel_new(raiden, event, current_block_number, chain_id)
```

Comment:
I believe this value should have been added to all state changes that are related to token networks and channels. Why is it only being used in `handle_channel_new`?